sda-download/api/api.md

```diff
@@ -19,13 +19,13 @@ The client can establish a session to bypass time-costly visa validations for fu
 
 ### Endpoints overview:
 
-**Data out API**:
+**[Data out API](#data-out-api)**:
 
 - `/metadata/datasets`
 - `/metadata/datasets/*dataset`
 - `/files/:fileid`
 
-**htsget**:
+**[File download requests, for htsget](#file-download-requests)**
 
 - `/s3/<datasetid>/<fileid>`
```

Comment:
To not confused with the `fileid` in the `/files` endpoint, I think it is better to use `/s3/<datasetid>/<filepath>`. Similar to the `/s3-encrypted` endpoint.

Reply:
Good point. How about this (using the same syntax as for the data out API endpoints):
```
- `/s3/*datasetid/*filepath`
- `/s3-encrypted/*datasetid/*filepath`
```